src/Module/Api/Mastodon/Instance.php

```diff
@@ -22,6 +22,7 @@
 namespace Friendica\Module\Api\Mastodon;
 
 use Friendica\Core\System;
+use Friendica\DI;
```

Comment:
Please use DI directly per constructor :)

Reply:
I do not follow you, the `BaseApi` constructor doesn't have `DI` parameter?

Reply:
No, sry what I mean is that you can replace all `DI::*` calls with a (maybe extended) parameter of the `__constructor()` of this method, so you don't need this import at all
--> we're already inside the DI, so no extra DI call necessary :)

Reply:
Ah, I get it now, thanks.